src/bundles/ApplicationsAdmin/components/AppList/AppList.js

```diff
@@ -73,7 +73,8 @@ class AppList extends Component {
             </div>
             <div className="col-sm-4 col-xs-12">
               <label htmlFor="keyword">Search:</label>
-              <input id="keyword" type="text" size="30" placeholder="id or name" onChange={onKeywordChange}/>
+              <input id="keyword" type="text" size="30" onChange={onKeywordChange}/>
+              <p style={{'font-size':'10px'}}>Search by: Profile Name, Application ID, Email Domain</p>
```

Comment:
Can you please increase the font size to 14px?  We should also create a class in AppList.css with these styles rather than inline them.

Reply:
Increased font size to 13px as Resh suggessted

Reply:
removed inline style and increased font-size to 14